azure_functions_worker/logging.py

```diff
@@ -51,25 +59,19 @@ def setup(log_level, log_destination):
 
 
 def disable_console_logging() -> None:
+    # We should only remove the sys.stdout stream, as error_logger is used for
+    # unexpected critical error logs handling.
     if logger and handler:
+        handler.flush()
         logger.removeHandler(handler)
 
-    if error_logger and error_handler:
-        error_logger.removeHandler(error_handler)
-
```

Comment:
So we are removing the two different handlers - `handler` and `error_handler`?

Reply:
Previously, we removed error_handler. But when I tested today, I think we should preserve error_logger, since sys.stderr will always picked up by host. This PR here is to stop removing error_handler.